build.go

```diff
@@ -88,6 +88,7 @@ func Build(
 		cachedChecksum, ok := bundlerLayer.Metadata[DepKey].(string)
 
 		dependencyChecksum := dependency.Checksum
+		//nolint Ignore SA1019, informed usage of deprecated field
```

Comment:
Any specific reason you went with a linting exception rather than switching to use the `Checksum` field?

Reply:
@sophiewigmore ^^

Reply:
Yea I guess we could remove support for the `SHA256` field at this point. I didn't want to just rip out support, but now that I think about it none of the versions we support even have that field so probably OK to remove it. I will do that 👍